components/SearchSandbox/components/InputRenderer.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
```

Comment:
You should move all these inputs in separate individual component files.

Reply:
Yeah will make separate components. Is it good to go ? Should I make this changes in other files too ?